pkg/controller/storagecluster/uninstall_reconciler.go

```diff
@@ -593,3 +600,28 @@ func (r *ReconcileStorageCluster) deleteSnapshotClasses(instance *ocsv1.StorageC
 	}
 	return nil
 }
+
+func (r *ReconcileStorageCluster) deleteQuickStarts(sc *ocsv1.StorageCluster, reqLogger logr.Logger) error {
+	for _, qs := range AllQuickStarts {
+		cqs := consolev1.ConsoleQuickStart{}
+		err := yaml.Unmarshal(qs, &cqs)
+		if err != nil {
+			reqLogger.Info(fmt.Sprintf("failed to unmarshal consolequickstarts, name: %s, namespace: %s", cqs.Name, cqs.Namespace))
+			return err
+		}
+		found := consolev1.ConsoleQuickStart{}
+		err = r.client.Get(context.TODO(), types.NamespacedName{Name: cqs.Name, Namespace: cqs.Namespace}, &found)
+		if err != nil {
+			if errors.IsNotFound(err) {
+				return nil
+			}
+			return err
+		}
+		err = r.client.Delete(context.TODO(), &found)
+		reqLogger.Info("Uninstall: Deleting QuickStart %s", "QuickStart Name", cqs.Name)
```

Comment:
Log this message only if the deletion succeeds - it is unnecessary to log both "deleting" and "failed to delete" on failure.

Reply:
done